engines/wintermute/base/sound/base_sound_manager.cpp

```diff
@@ -97,6 +97,12 @@ BaseSoundBuffer *BaseSoundMgr::addSound(const Common::String &filename, Audio::M
 		return nullptr;
 	}
 
+	if (filename.empty()) {
+		// At least one game, Bickadoodle, calls playSound with an empty filename, see #6594
+		BaseEngine::LOG(0, "addSound called with empty filename");
```

Comment:
Are you sure that it actually does that in the scripts (and i.e. we don't screw something up along the way?)

Does WME proper allow for calling with invalid/blank filenames?